kubetest2/internal/deployers/eksapi/templates/unmanaged-nodegroup-efa.yaml.template

```diff
@@ -317,8 +317,10 @@ Resources:
     Properties:
       AutoScalingGroupName: !Ref ResourceId
       MixedInstancesPolicy:
+        {{- if .Features.InstancesDistribution }}
```

Comment:
do we even need the `InstancedDistribution`? I just copied this from an unmanaged nodegroup example to begin with, wr can probably just remove it entirely if it's not available everywhere.

*or* we can just use a CFN intrinsic func to switch this off in certain partitions, if that's what we're getting at. Would be easier than pushing that logic up to the user of the deployer IMO

Reply:
in that case i think i'd rather pull it out because it doesn't seem required 🤔 

not sure if the parameters will affect runtimes based on capacity of Spot vs OnDemand, but the instance type prioritization seems like the only real different behavior

Reply:
ack, can you test and make sure the nodes still come up without this?

Reply:
ran a test successfully and updated the overview with the steps